app/lib/router/general_router.dart

```diff
@@ -264,6 +265,16 @@ final generalRoutes = [
           );
     },
   ),
+
+  GoRoute(
```

Comment:
we need to if-guard this as well to prevent `mocktail` from being included in the release build.

```suggestion
   if (includeChatShowcase)
  GoRoute(
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/2783/commits/3bc21f991f25045fabe97b167ff89051ff77a68b